aws/PAS-AIO-DR-template.yaml

```diff
@@ -0,0 +1,1227 @@
+                powershell -Command "& {&'Import-Module' WebAdministration}"; "&
+                {&'Start-WebAppPool' -Name PasswordVaultWebAccessPool}"; "&
+                {&'Set-ItemProperty' -Path
+                IIS:\AppPools\PasswordVaultWebAccessPool -Name autoStart -Value
+                'true'}"
+              waitAfterCompletion: '0'
+            105-CSTserviceConfig:
+              command: sc config "CyberArk Scheduled Tasks" start=auto
+              waitAfterCompletion: '0'
+        configCPM:  
+          commands:
+            201-configuration:
+              command: !Join
+                - ' ' 
+                - 
+                  - powershell.exe -File C:\CyberArk\Deployment\CPMConfiguration.ps1 -VaultIpAddress
+                  - !GetAtt 'VaultMachine.PrivateIp'
+                  - -VaultAdminUser "Administrator" -VaultPort 1858
+              waitAfterCompletion: '0'
+            202-registration:
+              command: !Sub >
+                powershell.exe -File C:\CyberArk\Deployment\CPMRegistration.ps1
+                -VaultAdminUser "Administrator" -SSMAdminPassParameterID
+                ${StoreAdminPassword.SsmId}
+              waitAfterCompletion: '0'
+            203-configureCPMService:
+              command: powershell.exe -File C:\CyberArk\Deployment\Set-LocalService.ps1 -Component 'CPM' -Username "PasswordManagerUser" -Services "CyberArk Central Policy Manager Scanner"
+              waitAfterCompletion: '0'
+            204-configureCPMService:
+              command: powershell.exe -File C:\CyberArk\Deployment\Set-LocalService.ps1 -Component 'CPM' -Username "PasswordManagerUser" -Services "CyberArk Password Manager"
+              waitAfterCompletion: '0'
+            205-CPMserviceConfig:
+              command: sc config "CyberArk Password Manager" start=auto
+              waitAfterCompletion: '0'
+            206-CPMSserviceConfig:
+              command: sc config "CyberArk Central Policy Manager Scanner" start=auto
+              waitAfterCompletion: '0'
+        configPSM:  
+          commands:
+            301-configuration:
+              command: !Join
+                - ' ' 
+                -  
+                  - powershell.exe -File C:\CyberArk\Deployment\PSMConfiguration.ps1 -VaultIpAddress
+                  - !GetAtt 'VaultMachine.PrivateIp' 
+                  - -VaultAdminUser "Administrator" -VaultPort 1858
+              waitAfterCompletion: '0'
+            303-registration:
+              command: !Sub >
+                  powershell.exe -File C:\CyberArk\Deployment\PSMRegistration.ps1
+                  -VaultAdminUser "Administrator" -SSMAdminPassParameterID ${StoreAdminPassword.SsmId}
+              waitAfterCompletion: '0'  
+            304-PSMserviceConfig:
+              command: sc config "Cyber-Ark Privileged Session Manager" start=auto
+              waitAfterCompletion: '0'
+        configSignal:  
+          commands:
+            98-SignalCompletion:
```

Comment:
first restart the instance, and then signal for completion
change the order of commands

Reply:
changed the order